service/measurement-server/README.md

```diff
@@ -0,0 +1,127 @@
+# Service: CCNP Measurement Server
+
+Measurements are key assets in the confidential computing world for user to verify and determine the trustworthiness of the environment and its underlying platform.
+Users can utilize different kinds of measurements to monitor or validate the operations performed within the platform.
+
+This service (which is actually a GRPC server) will provide support to fetch the measurements for confidential cloud native environments, including TDX RTMR measurements, TPM PCR measurements and TEE reports.
+
+
+
+## Introduction
+
+This service provides functionality to fetch measurements for confidential cloud native environments, both platform level(`PAAS` option) and container level (`SAAS` option). Using this service, user can fetch different
+categories of measurements including: TDX RTMR measurements (`TDX_RTMR` option), TPM PCR measurements (`TPM` option) and TEE reports from different vendors (`TEE_REPORT` option).
+Here shows the proto buf for the service:
+
+```
+enum TYPE {
+    PAAS = 0;
+    SAAS = 1;
+}
+
+enum CATEGORY {
+    TEE_REPORT = 0;
+    TPM = 1;
+    TDX_RTMR = 2;
+}
+
+message GetMeasurementRequest {
+    TYPE measurement_type = 1;
+    CATEGORY measurement_category = 2;
+    string report_data = 3;
+    int32 register_index = 4;
+
+}
+
+message GetMeasurementReply {
+    string measurement = 1;
+}
+
+service Measurement {
+    rpc GetMeasurement (GetMeasurementRequest) returns (GetMeasurementReply) {}
+}
+```
+
+To select different level and category of measurements, user shall send out request with different settings.
+User can find sample request in the Testing section.
+
+The collected measurements are returned as json string to the client.
+
+
+
+## Installation
+
+The Measurement service can be deployed as either DaemonSet or sidecar in different user scenarios.
```

Comment:
according to different user scenario???

Reply:
The user scenario here means if they are using CCNP for confidential cluster(deploy as Daemonset) or confidential containers(deploy as sidecar), is it appropriate to use 'user scenario' to differentiate such two cases?

Reply:
fine